data_steward/tools/run_curation_pipeline.sh

```diff
@@ -14,6 +14,10 @@ Usage: run_curation_pipeline.sh
   --dataset_release_tag <release tag for the CDR>
   --ehr_cutoff <ehr_cut_off date format yyyy-mm-dd>
   --rdr_export_date <date RDR export is run format yyyy-mm-dd>
+  --ticket_number <Ticket number to append to sandbox table names>
```

Comment:
you may want to mention this is to remove deactivated participants.

Reply:
I'll make this change, and push it in a new PR.